NEWS.md

```diff
@@ -0,0 +1,30 @@
+# Release notes for xrdp v0.9.1 (2016/12/21)
+## Added
+  * New xorgxrdp backend using existing Xorg with additional modules
+  * Improvements to X11rdp backend
+  * Support for IPv6 (disabled by default)
+  * Initial support for RemoteFX Codec (disabled by default)
+  * Support for multiple security layers (RDP and TLS).
+  * Support for disabling deprecated ssl versions (like SSLv3), and also select custom cipher suites in xrdp.ini
+  * Support for bi-directional fastpath
+  * Support clients that doesn't support drawing orders, such as MS RDP client for Android, ChromeRDP (disabled by default)
+  * More configurable login screen
+  * Support for several new channels:
+      * rdpdr: Device Redirection
+      * rdpsnd: Audio Output
+      * cliprdr: Clipboard
+      * xrdpvr: xrdp video redirection channel (can be used along with NeutrinoRDP client)
+  * Support for disabling channels globally or by session type
+  * Allow to specify the path for backends (Xorg, X11rdp, Xvnc)
+  * Added files for systemd support
+  * Multi monitor support
+
+## Changed
+* Transport: xrdp will prefer TLS transport when possible by client.
+* RDP: xrdp will do fastpath commands by default now.
+* xrdp-chansrv: chansrv logs are stored in `${XDG_DATA_HOME}/xrdp` now
```

Comment:
Please remove "chansrv", it's implied by "xrdp-chansrv". The whole section should have a space before "*" for consistency with other sections.